src/stores/index.js

```diff
@@ -59,6 +60,11 @@ const SSH_TUNNEL_FIELDS = [
  */
 const EXTENSION = 'Connect.Extension';
 
+const CONNECT_VIEWS = {
+  CONNECTION_STRING: 'connectionString',
```

Comment:
What about just `CONNECTION_STRING_VIEW` and `CONNECTION_FORM_VIEW`?

Reply:
sure I like that